reader.js

```diff
@@ -44,7 +51,7 @@ function respondToHTTPReq(request, response) {
 
         respondToIDsQuery(mission, response);
 
-    } else {
+    } else if (requestQuery.search.includes('?timestamps%5B%5D=') > 0) {
```

Comment:
There might be a cleaner way to do this. Additionally, it will break when `[]` is not URI-encoded. Maybe give it a separate path, eg `localhost:5250/MISSION_ID/data`?